test/e2e/etcd_backup_test.go

```diff
@@ -280,7 +280,7 @@ func deleteAndCheckEtcd(ctx context.Context, cl client.Client, logger logr.Logge
 	ExpectWithOffset(1,
 		cl.Get(
 			ctx,
-			client.ObjectKey{Name: "etcd-bootstrap-" + string(etcd.UID[:6]), Namespace: etcd.Namespace},
+			client.ObjectKey{Name: etcd.Name + "-config-" + string(etcd.UID[:6]), Namespace: etcd.Namespace},
```

Comment:
```suggestion
			client.ObjectKey{Name: "etcd-config-" + string(etcd.UID[:6]), Namespace: etcd.Namespace},
```

Reply:
Actually what he has done is right. Please check the naming convention in `v1alpha1.GetConfigMapName`